app/assets/stylesheets/survey_index.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+@import "foundation_and_overrides";
+
+
+
+.button.success.radius.small {
+  margin-left: 170px;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, border-width, color, font-family, font-size, margin-left, margin-right, padding-bottom, padding-left, padding-right, padding-top